frontend/src/component/pages/Home.tsx

```diff
@@ -290,6 +297,7 @@ export class Home extends Component<Props, State> {
             onLongLinkTextFieldChange={this.handlerLongLinkChange}
             onShortLinkTextFieldBlur={this.handlerCustomAliasTextFieldBlur}
```

Comment:
This is not related to you. We need to change `handler` to `handle` in the future for code consistency.

Reply:
TODO
- [ ] Change names of "handler" functions to say "handle" instead to be more consistent